### ISSUE TITLE
Kappa Everywhere not working in livestreaming beta

### ISSUE THREAD

**Issue by 7596ff** [popcorncolonel/Chrome-Extensions#8]
![ss 2015-07-29 at 10 58 58](https://cloud.githubusercontent.com/assets/1760581/8974924/6e0fc2c6-3645-11e5-8802-15305d537706.png)

Here's a screenshot to demonstrate it, but seems like while this may not be a simple fix, it's a relatively nice one to have.

**Comment by popcorncolonel** [comment]
Do you have a link to the site?

**Comment by brendan10211** [comment]
Just go to http://www.dota2.com/watch and click on one of the streams there.

**Comment by popcorncolonel** [comment]
Hmmm.... Looks like it might be because it's in an <iframe>? It's kind of hard to debug with so many chat messages coming from the site anyway.

**Comment by alasher** [comment]
I did a little prodding and found that DOMNodeInserted event doesn't run when a new chat message appears for the dota2 watch page. Don't know why. I know DOMNodeInserted is deprecated anyway, I think updating it to something newer could potentially fix the issue. Event if it doesn't, it's a step toward's future-proofing the extension.

I spent a little time working with MutationEvents but I haven't had luck yet. If I make any decent progress, I'll submit a pull request.
